docs/getting-started/overview.md

```diff
@@ -79,32 +75,39 @@ The following diagram illustrates the single point of access through the Gateway
+The Discovery Service is the central repository of active services in the API ML ecosystem. The Discovery Service 
+continuously collects and aggregates service information and serves as a repository of active services. When a service is 
+started, it sends its metadata, such as the original URL, assigned serviceId, and status information to the Discovery Service. 
+Back-end microservices register with this service either directly or by using a Eureka client. Multiple enablers are 
+available to help with service on-boarding of various application architectures including plain Java applications and 
+Java applications that use the Spring Boot framework. The Discovery Service is built on Eureka and Spring Boot technology.
 
 **API Catalog**
 
-The API Catalog is the catalog of published API services and their associated documentation. The Catalog provides both the REST APIs and a web user interface (UI) to access them. The web UI follows the industry standard Swagger UI component to visualize API documentation in OpenAPI JSON format for each service. A service can be implemented by one or more service instances, which provide exactly the same service for high-availability or scalability.
-
-**Catalog Security**
-
-Access to the API Catalog can be protected with an Enterprise z/OS Security Manager such as IBM RACF, ACF2, or Top Secret. Only users who provide proper mainframe credentials can access the Catalog. Client authentication is implemented through the z/OSMF API.
+The API Catalog is the catalog of published API services and their associated documentation. The Catalog provides both 
+the REST APIs and a web user interface (UI) to access them. The web UI follows the industry standard Swagger UI component 
+to visualize API documentation in OpenAPI JSON format for each service. A service can be implemented by one or more service 
+instances, which provide exactly the same service for high-availability or scalability. API Catalog requires authentication
+from the accessing user. 
 
 **Caching Service**
 
-An API is provided in high-availability mode which offers the possibility to store, retrieve, and delete data associated with keys. The service can only be used by internal Zowe services and is not exposed to the internet.
+An API is provided in high-availability mode which offers the possibility to store, retrieve, and delete data associated 
+with keys. 
 
 #### Onboarding APIs
-Essential to the API Mediation Layer ecosystem is the API services that expose their useful APIs. Use the following topics to discover more about adding new APIs to the API Mediation Layer and using the API Catalog:
+Essential to the API Mediation Layer ecosystem is the API services that expose their useful APIs. Use the following topics 
+to discover more about adding new APIs to the API Mediation Layer and using the API Catalog:
```

Comment:
Please change "discover" to "learn." "Discover" sounds a little awkward in this particular use.